tee-worker/core-primitives/substrate-sgx/externalities/src/lib.rs

```diff
@@ -175,6 +181,41 @@ where
 		self.state_diff.clear();
 	}
 
+	// Note: This implementation only works for keys encoded with Blake2_128Concat
+	fn iter_prefix<K: Decode + Debug, V: Decode + Debug>(
+		&self,
+		key_prefix: &[u8],
+	) -> Option<Vec<(K, V)>> {
```

Comment:
I find it simpler to return just empty vector instead of `Option` in such cases.

Reply:
I prefer `Option` here, The other functions also use `Option` and returning an empty vector is not very ergonomic.